server/bundler/loader.js

```diff
@@ -84,7 +83,7 @@ function splitTemplate( path, section ) {
 		'page( ' + pathRegex + ', function( context, next ) {',
 		'	if ( _loadedSections[ ' + JSON.stringify( section.module ) + ' ] ) {',
 		'		controller.setSection( ' + JSON.stringify( section ) + ' )( context );',
-		'		layoutFocus.next();',
+		'		context.store.dispatch( { type: "LAYOUT_NEXT_FOCUS_ACTIVATE" } );',
```

Comment:
>I was led astray

Yeah, noticed that too while going back through history. We should update those as well separately.